platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/Level.java

```diff
@@ -84,4 +84,22 @@ public static Level valueOfOrElse(@Nullable final String value, @NonNull final L
             return defaultLevel;
         }
     }
+
+    public String nameWithFixedSize() {
```

Comment:
nonnull + javadoc

Reply:
As we discussed, we are addressing this in a different PR and marking this as resolved to merge this one.